backend/graphql/resolvers/MsgComment.js

```diff
@@ -0,0 +1,47 @@
+const MsgComment = require('../../models/MsgComment');
+
+const msgCommentResolvers = {
+	Query: {
+		MsgComments: () => MsgComment.find().populate('user message likes'),
+		findMsgComment: (_, { input: { id } }) => {
+			return MsgComment.findById(id).populate('user message likes');
+		}
+	},
+	Mutation: {
+		addMsgComment: (_, { input }) => {
+			const { user, message, content } = input;
+			if (!user || !message || !content)
+				throw new Error('User, message and content required.');
+			return new MsgComment(input)
+				.save()
+				.then(comment => comment.populate('user message likes').execPopulate());
+		},
+		updateMsgComment: (_, { input }) => {
+			const { id, user, message, content } = input;
+			if (!id && !user && !message && !content)
+				throw new Error('No user, message or content provided');
```

Comment:
Given the content of the `if` logic check above, this thrown error might be confusing if the update is only missing an ID for some inexplicable reason.